### PR TITLE
Loading views and state

### PR DESCRIPTION
**Pull request by PedroCarrillo**
#46

### CODE REVIEW

File: app/src/main/res/values/no_data_found.xml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources></resources>
```
Comment (cpinan) [inline app/src/main/res/values/no_data_found.xml]: Any reason why this file still exist?

Comment (PedroCarrillo) [inline app/src/main/res/values/no_data_found.xml]: if you don't see the importance then we are in trouble.

Comment (gyosida) [inline app/src/main/res/values/no_data_found.xml]: lol delete it pls

**Comment by gyosida** [comment]
close #46

**Pull request merged**